src/components/About/styles.scss

```diff
@@ -0,0 +1,148 @@
+// Mobile portrait
+.wrapper {
+	display: flex;
+	width: 100%;
+	height: 100%;
+	flex-direction: column;
+	background-color: blue;
+}
+
+	.hero {
+		display: block;
+		width: 100%;
+		height: 38%;
+	}
+
+	.textContainer {
+		width: 100%;
+		height: 62%;
+		--padding__body: 10vw;
+		padding: var(--padding__body) var(--padding__body) 0 var(--padding__body);
+		overflow-y: auto;
+		font-size: var(--font_size__body);
+		background-color: pink;
+	}
+
+		.heading, .subheading {
+			font-family: var(--font_family__heading);
+		}
+
+		.heading {
+			margin-bottom: 0.5rem;
+			font-size: var(--font_size__heading);
+		}
+
+		.subheading {
+			width: 85%;
+			margin-bottom: 1rem;
+			font-size: var(--font_size__subheading);
+			line-height: 1.5em;
+		}
+		
+		.body {
+			margin-bottom: var(--padding__body);
+			font-family: var(--font_family__body);
+			line-height: 1.5em;
+			background-color: green;
+
+			p + p {
+				margin-top: 1em;
+			}
+		}
+
+// Mobile landscape
+@media (orientation: landscape) {
+	.wrapper {
+		flex-direction: row;
+	}
+
+		.hero {
+			width: 50%;
+			height: 100%;
+		}
+
+		.textContainer {
+			width: 50%;
+			height: 100%;
+			--padding__body: 5vw;
+		}
+}
+
+// Tablet portrait
+@media (min-width: 768px) {
+	.hero {
+		height: 60%;
+	}
+
+	.textContainer {
+		height: 40%;
+		--font_size__factor: 1.62;
+		font-size: calc(var(--font_size__body)*var(--font_size__factor));
+	}
+
+		.heading {
+			font-size: calc(var(--font_size__heading)*var(--font_size__factor));
```

Comment:
yerr, I can see why this felt weird - this is definitely where `em` comes in instead of `rem` - this is fine for now, but we should maybe take a look at this at some point together to find a neat solution for it.